imu_node:
  ros__parameters:
    timer_tick: 0.03
    imu_enable: 1